CityAssistant/CityAssistant/Views/CityTourView.swift

```diff
@@ -42,14 +68,14 @@ struct CityTourView: View {
     
     init(pois: [POI]) {
         viewModel = CityTourViewModel(pois: pois)
-        poiViewModel = POIViewModel()
+        poiViewModel = POIViewModel(business: CityTourView_Previews.mockPOI)
```

Comment:
I get why you're doing this but I think a better solution would be to just initialize CityTour with the pois that come out of the async content wrapper. Not really important since this initializer is only used by the previews.